superset/assets/src/featureFlags.ts

```diff
@@ -21,6 +21,7 @@
 export enum FeatureFlag {
   SCOPED_FILTER = 'SCOPED_FILTER',
   OMNIBAR = 'OMNIBAR',
+  CLIENT_CACHE = 'CLIENT_CACHE',
```

Comment:
is CLIENT_CACHE feature enabled by default or not?

Reply:
It is not. If it's not present in `config.py` it's treated as false.

Reply:
thanks! then dashboard's integration tests are based on default behavior, POST requests. I think you probably need to revert tests changes from #7032 ?

Reply:
Right, doing it now :)